base/compiler/typeinfer.jl

```diff
@@ -45,6 +45,8 @@ function _typeinf_identifier(frame::Core.Compiler.InferenceState)
     return mi_info
 end
 
+_typeinf_identifier(frame::InferenceFrameInfo) = frame
+
```

Comment:
Unrelated change?

Reply:
No, it gets used here: https://github.com/JuliaLang/julia/pull/44803/files#diff-cf06383638884136da67baacb3cec2ed1b9cf52ac3443f783a999a58184f890aR389-R395